app/addons/activetasks/components.react.jsx

```diff
@@ -221,7 +218,7 @@ var ActiveTasksTableHeader = React.createClass({
     });
   },
 
-  render: function () {
+  render () {
```

Comment:
i'm a fan of separating refactor step from the actual commit as they clutter the diff, but i think i lost that fight already long time ago haha